RecoParticleFlow/PFProducer/python/mlpfproducer_customise.py

```diff
@@ -0,0 +1,48 @@
+import FWCore.ParameterSet.Config as cms
+
+def customise_step3(process, output="AODSIMoutput", usegpu=False):
+    if hasattr(process, "particleFlowBlock") and hasattr(process, output):
+        process.mlpfproducer = cms.EDProducer("MLPFProducer",
+            #we don't really need anything the block does, just to get all the PFElements in the event in a convenient list
+            src=cms.InputTag("particleFlowBlock"), 
+            model_path=cms.string("RecoParticleFlow/PFProducer/data/mlpf/mlpf_2020_11_04.pb")
+        )
+        process.ak4MLPFJets = process.ak4PFJets.clone()
+        process.ak4MLPFJets.src = cms.InputTag("mlpfproducer")
+
+        out = getattr(process, output)
+        out.outputCommands.append('keep recoPFCandidates_*_*_*')
+        out.outputCommands.append('keep *_ak4MLPFJets_*_*')
+        out.outputCommands.append('keep *_ak4PFJets_*_*')
+        process.mlpf_path = cms.Path(process.mlpfproducer*process.ak4MLPFJets)
+
+        if usegpu:
```

Comment:
a few notes:
* Triton can also run on CPU
* in the hopefully near future, a ProcessModifier for SonicTriton will be introduced; then this can be enabled more automatically

Reply:
Right. Once a Triton-specific or GPU-specific modifier is available, this should be adapted accordingly.

Reply:
presumably architecture-specific stuff needs to be done via fwk mechanisms to keep the pset agnostic of the hardware used. I guess ProcessModifiers won't fit that model?

Reply:
I'm not sure what is the best way of signaling to the fwk that a GPU (or other accelerator, and/or an external service provider) should be used, but I'm happy to adapt if there's a way! This customization function is in any case mainly for testing.

Reply:
@davidlange6 I'm planning to introduce a Service to keep track of whether remote GPU servers are available. For the time being, the ProcessModifier will enable the Service and swap in the relevant Triton producers.

Reply:
I've removed the customization functions and introduced an `mlpf` modifier, which for the moment only enables the TF-cpu evaluator. Once we have an established method of enabling triton & running tests with it, it can be added here.